src/components/BreadCrumb/BreadCrumb.component.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import './BreadCrumb.styles.scss';
+import { useAuth } from '@providers/Auth';
+
+function BreadCrumb(props) {
+  const { authenticated } = useAuth();
+
+  return (
+    <ul className={'breadcrumb ' + props.className}>
+      <li
+        className={
+          'breadcrumb__item ' +
+          (props.selected === 'related' ? 'breadcrumb__item--selected' : '')
+        }
+        onClick={() => props.onSelectedChange('related')}
+      >
+        <h5>Related</h5>
+      </li>
+      {authenticated ? (
+        <li
+          className={
+            'breadcrumb__item ' +
+            (props.selected === 'favorites' ? 'breadcrumb__item--selected' : '')
+          }
+          onClick={() => props.onSelectedChange('favorites')}
+        >
+          <h5>Favorites</h5>
+        </li>
+      ) : null}
+    </ul>
+  );
+}
+
+BreadCrumb.defaultProps = {
+  className: '',
```

Comment:
Good practice, setting the default props to avoid type errors on runtime.
I think you don't need to set className default prop to `''`.
React DOM doesn't set the `class` attribute for the element with an `undefined` value, but we can discuss this further.